pkg/services/cloudmigration/cloudmigrationimpl/gcomstub.go

```diff
@@ -49,6 +49,10 @@ func (client *gcomStub) ListAccessPolicies(ctx context.Context, params gcom.List
 	return items, nil
 }
 
+func (client *gcomStub) ListTokens(ctx context.Context, params gcom.ListTokenParams) ([]gcom.TokenView, error) {
+	panic("unimplemented")
```

Comment:
I think we should implement this to make the frontend task smoother. IMO we should just have...

- the `CreateToken()` func set a `*gcom.Token` on `client`
- the `ListTokens()` func return it as a `TokenView` if it exists; if it's nil, return `ErrTokenNotFound`